diesel/src/sqlite/expression/functions.rs

```diff
@@ -892,4 +892,198 @@ extern "SQL" {
         j: J,
         path: Text,
     ) -> Nullable<Text>;
+
+    /// The json_extract(X,P1,P2,...) extracts and returns one or more values from the well-formed JSON at X.
+    ///
+    /// If only a single path P1 is provided, then the SQL datatype of the result is NULL for a JSON null,
+    /// INTEGER or REAL for a JSON numeric value, an INTEGER zero for a JSON false value, an INTEGER one for a JSON true value,
+    /// the dequoted text for a JSON string value, and a text representation for JSON object and array values.
+    ///
+    /// If there are multiple path arguments (P1, P2, and so forth) then this routine returns SQLite text
+    /// which is a well-formed JSON array holding the various values.
```

Comment:
This part is not possible with the function signature below. This refers to a use case where you would call `json_extract(json_col, path_1, path_2)` which is disallowed by the signature.